castoredc_api/client/castoredc_api_client.py

```diff
@@ -41,18 +44,29 @@ def __init__(self, client_id, client_secret, url):
         self.base_url = f"https://{url}/api"
         self.auth_url = f"https://{url}/oauth/token"
 
+        try:
+            self.package_version = pkg_metadata.version("castoredc_api")
+        except pkg_metadata.PackageNotFoundError:
+            self.package_version = "Github Tests"
```

Comment:
I would probably use:

```suggestion
        try:
            self.package_version = pkg_metadata.version("castoredc_api")
        except pkg_metadata.PackageNotFoundError:
            self.package_version = "unknown"
```

Or even:

```suggestion
        try:
            self.package_version = pkg_metadata.version("castoredc_api")
        except pkg_metadata.PackageNotFoundError:
            self.package_version = "dev"
```

Reply:
Changed this to your suggestion